src/helpers/tweet/split-tweet-text/split-tweet-text.ts

```diff
@@ -24,19 +26,40 @@ const splitTweetText = async (
     mastodonUsername,
   );
 
+  let tweetText = text!;
+
+  if (mentions != null) {
+    for (const tweetMention of mentions) {
+      const mapped = mentionsMapping.find(
+        (mapping) => tweetMention.username == mapping.twitter,
+      );
+      if (mapped == null) {
+        continue;
+      }
+      const newMention =
+        platform === Platform.MASTODON ? mapped!.mastodon : mapped!.bluesky;
+      if (newMention != null) {
+        tweetText = tweetText.replaceAll(
+          `@${tweetMention.username}`,
+          `@${newMention}`,
+        );
+      }
+    }
+  }
+
```

Comment:
I'm on my phone, so this comment won't be that helpful, but my feeling is that conditions and typing are a bit too complicated /flacky (but again, I don't have the ide to actually see all the typings)